File: video/rooms/participants/retrieve-participant/retrieve-a-participant.java
import com.twilio.Twilio;
import com.twilio.rest.video.v1.room.Participant;
public class ParticipantsExample1 {
	public static final String API_KEY_SID = "SKXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
	public static final String API_KEY_SECRET = "your_api_key_secret";
	
	public static void main(String args[]) {
		Twilio.init(API_KEY_SID, API_KEY_SECRET);
		
		Participant participant = Participant.fetcher("RMXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX", "Alice").fetch();
	}
}

File: video/rest/rooms/participants/retrieve-participant/retrieve-a-participant.php
<?php
// Get the PHP helper library from twilio.com/docs/php/install
require __DIR__ . '/twilio-php-main/Twilio/autoload.php';  // Loads the library
use Twilio\Rest\Client;

// Your Account Sid and Auth Token from twilio.com/console
$apiKeySid = "SKXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
$apiKeySecret = "your_api_key_secret";
$client = new Client($apiKeySid, $apiKeySecret);

$participant = $client->video->rooms("DailyStandup")
      ->participants("Alice")
      ->fetch();

echo $participant->duration;
